main.py

```diff
@@ -142,7 +156,15 @@ def api_template(my_path=''):
         if UPLOAD_LIMIT is not None and CURRENT_UPLOAD_SIZE >= UPLOAD_LIMIT:
             return Response(status=400, response='Reached upload limit')
         with open(os.path.join('static/data/', my_path,  'template.json'), "w+") as outfile:
+            data = request.get_json()
             json.dump(request.get_json(), outfile, sort_keys=True, indent=4)
+        # Open template file to make sure what we just created matches what was submitted.
+        template = open(os.path.join('static/data/',(my_path + 'template.json')))
```

Comment:
no guarantee that this will exist and it should use the same recursive backwards search as the get template function